pkg/api/handlers/compat/containers.go

```diff
@@ -463,9 +463,20 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*container.InspectResp
 		state.Running = true
 	}
 
-	// Dockers created state is our configured state
-	if state.Status == define.ContainerStateCreated.String() {
-		state.Status = define.ContainerStateConfigured.String()
+	// map our statuses to Docker's statuses
+	switch state.Status {
+	case define.ContainerStateConfigured.String(), define.ContainerStateCreated.String():
+		state.Status = "created"
+	case define.ContainerStateRunning.String(), define.ContainerStateStopping.String():
+		state.Status = "running"
```

Comment:
I wasn't 100% sure what to do with the "Stopping" state, docker doesn't have that state, and [the conversion in `LibpodToContainer`](https://github.com/containers/podman/blob/923501c24facb64975f9c672540efbcd7f43e961/pkg/api/handlers/compat/containers.go#L330-L331) just leaves it as "Stopping". This seemed like the most correct place to put it.